include/mme-app/mmeSmDefs.h

```diff
@@ -41,25 +41,31 @@ const uint16_t detach_start = 17;
 const uint16_t detach_wf_del_session_resp = 18;     
 const uint16_t detach_wf_purge_resp = 19;     
 const uint16_t detach_wf_purge_resp_del_session_resp = 20;     
-const uint16_t ni_detach_start = 21;     
-const uint16_t ni_detach_wf_del_sess_resp = 22;     
-const uint16_t ni_detach_wf_det_accpt_del_sess_resp = 23;     
-const uint16_t ni_detach_wf_detach_accept = 24;     
-const uint16_t ni_detach_wf_s1_rel_comp = 25;     
-const uint16_t paging_start = 26;     
-const uint16_t paging_wf_service_req = 27;     
-const uint16_t s1_release_start = 28;     
-const uint16_t s1_release_wf_release_access_bearer_resp = 29;     
-const uint16_t s1_release_wf_ue_ctxt_release_comp = 30;     
-const uint16_t service_request_start = 31;     
-const uint16_t service_request_wf_aia = 32;     
-const uint16_t service_request_wf_auth_resp_validate = 33;     
-const uint16_t service_request_wf_auth_response = 34;     
-const uint16_t service_request_wf_init_ctxt_resp = 35;     
-const uint16_t service_request_wf_mb_resp = 36;     
-const uint16_t service_request_wf_sec_cmp = 37;     
-const uint16_t tau_start = 38;
-const uint16_t END_STATE = 39;
+const uint16_t intra_s1_ho_start = 21;     
```

Comment:
what is the reason to add something in between ? If we are expecting more things to come in between then let's keep hole ?

Reply:
This is an auto generated file. It just creates the constants in the alphabetical order. Kept it this way as its just easier to search if its in alphabetical order.